django/thunderstore/api/cyberstorm/views/package_listing.py

```diff
@@ -56,6 +58,13 @@ def get_description(self, obj: PackageVersion) -> str:
     def get_icon_url(self, obj: PackageVersion) -> Optional[str]:
         return obj.icon.url if obj.is_effectively_active else None
 
+    def get_is_unavailable(self, obj: PackageVersion) -> bool:
+        community = Community.objects.filter(
+            identifier=obj.community_identifier
+        ).first()
+
+        return obj.is_unavailable(community)
```

Comment:
Only noticed this on a second look, but the implementation of this seems hazardous for performance when sets of items are being rendered. I'd try to find a way to e.g. annotate this information to the queryset as a batch operation instead.